src/tools/OptionsSchema.json

```diff
@@ -386,6 +386,10 @@
           "type": "string",
           "description": "When set, text that the target application writes to stdout and stderr (ex: Console.WriteLine) will be saved to the specified file. This option is ignored if console is set to something other than internalConsole.",
           "default": "${workspaceFolder}/out.txt"
+        },
+        "targetArchitecture": {
+          "type": "string",
+          "description": "The architecture of the debuggee. This will automatically be detected unless this parameter is set. Allowed values are x86_64 or arm64."
```

Comment:
>arm64 [](start = 155, length = 5)

I would add a note that this is currently only supported for local debugging on macOS.